crates/query-engine/translation/src/translation/query/sorting.rs

```diff
@@ -97,11 +97,15 @@ struct Column(models::FieldName);
 /// An aggregate operation to select from a table used in an order by.
 #[derive(Debug)]
 enum Aggregate {
-    CountStarAggregate,
-    SingleColumnAggregate {
+    StarCount,
```

Comment:
These new names are better, but again, this change is arbitrary and just more noise, this kind of thing should just be a tiny gardening PR.